### ISSUE TITLE
Bandwidth limiter

### ISSUE THREAD

**Issue by kseistrup** [carlmjohnson/heffalump#1]
It would be nice if heffalump had a builtin bandwidth limiter so that output rate could be limited to a given number of kB per second.

**Comment by earthboundkid** [comment]
Yes. I think for now the solution is to put it behind Nginx or similar, and set the bandwidth limits there. From what I can tell, setting all the right timeouts for a production Go server [can be a bit complicated](https://blog.gopheracademy.com/advent-2016/exposing-go-on-the-internet/).

It might also be possible to add streaming GZIP through Nginx, although I'm not sure about that.
